docs/admin/addons.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
+
+This page lists some of the available add-ons and links to their respective installation instructions.
+
+## Networking and Network Policy
```

Comment:
I'd be happy to try and make the sentences feel less like marketing speak, but I feel strongly that giving the user a description of what each thing is makes for a much better UX than forcing them to click through to each page, which would be the case if it was a bare list of project names. (Remember, to a new user, the projects' names themselves will likely be meaningless.)